app/views/user_sessions/security_notification.html.haml

```diff
@@ -0,0 +1,31 @@
+= content_for :title, 'Search.gov Login'
+= content_for :navigation do
+  %ul.nav.signup
+    %li= link_to 'Sign up', signup_path
```

Comment:
What is the purpose of including the "Sign up" link on this page? That link goes to the old signup form, which I assume we'll be removing. Won't the signup path and the login path eventually be the same, taking the user to this page?

Reply:
So i'm following what we currently have.  If a user goes to the /login on production you will see the Sign up link there.  The path will eventually be redirected. But for now just focusing on getting the system notification out there.

Reply:
But won't it eventually be redirected to the same page? I'm wondering what the purpose is of adding it now if we're just going to remove it later.

Reply:
It was not added. This was the /login page and it is there currently on production. I'll focus on that when I get to that Jira ticket.